apps/native/app/src/ui/lib/alert/alert.tsx

```diff
@@ -85,6 +99,11 @@ const Icon = styled.View`
   align-self: flex-start;
 `
 
+const Content = styled.View`
+  padding-right: ${({ theme }) => theme.spacing[2]}px;
```

Comment:
~Changing this from previously 34px to 16px, is that intentional?

Reply:
I thought this was intentional. But after looking better at it I could not see that is would make any difference. So I took the spacing from figma instead